apps/deploy-web/env/.env.staging-testnet

```
@@ -0,0 +1,44 @@
+NEXT_PUBLIC_DEFAULT_NETWORK_ID=testnet
+NEXT_PUBLIC_BILLING_ENABLED=true
+NEXT_PUBLIC_AUTO_TOP_UP_ENABLED=true
+NEXT_PUBLIC_MANAGED_WALLET_NETWORK_ID=testnet
+NEXT_PUBLIC_MANAGED_WALLET_DENOM=uakt
+NEXT_PUBLIC_MASTER_WALLET_ADDRESS=akash1ss0d2yw38r6e7ew8ndye9h7kg62sem36zak4d5
+NEXT_PUBLIC_UAKT_TOP_UP_MASTER_WALLET_ADDRESS=akash13098gsk69v8tusyzhh24kl7d0fqxd4tzu7ng9h
+NEXT_PUBLIC_USDC_TOP_UP_MASTER_WALLET_ADDRESS=akash1g3teu8mqmg6ej79t34ev8uqk0dxq6kxl50nar3
+
+NEXT_PUBLIC_STATS_APP_URL=https://stats.akash.network
+
+NEXT_PUBLIC_SENTRY_ENABLED="true"
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)